galter_subjects_utils/cli.py

```diff
@@ -52,36 +47,71 @@ def to_mesh_downloader_kwargs(parameters):
 def to_mesh_converter_kwargs(parameters, downloader):
     """Convert."""
     result = {}
+
     if parameters["filter"] in ["topic", "topic-qualifier"]:
         result["topics"] = MeSHReader(
-            downloader.prefix_to_filepath["d"],
+            downloader.terms_filepath,
             filter=topic_filter,
         )
     if parameters["filter"] in ["topic-qualifier"]:
-        result["qualifiers"] = MeSHReader(downloader.prefix_to_filepath["q"])
+        result["qualifiers"] = MeSHReader(downloader.qualifiers_filepath)
 
     return result
 
 
-@main.command()
-@click.option("--year", "-y", default=defaults["year"])
-@click.option(
-    "--filter", "-f",
-    type=click.Choice(["topic", "topic-qualifier"]),
-    default=defaults["filter"],
-)
-@click.option(
-    "--downloads-dir", "-d",
-    type=click.Path(path_type=Path),
-    default=defaults["downloads-dir"])
+def mesh_download_options(f):
```

Comment:
Nice! 👍 

It's interesting to see someone skillfully use decorators. Definitely one one of the Python features that I struggle with.